wiki/Game_modifier/Hidden/id.md

```diff
@@ -0,0 +1,81 @@
+---
+stub: true
+tags:
+  - hidden
+  - HD
+  - mod
+  - game modifier
+---
+
+# Hidden (mod)
+
+![HD mod icon](/wiki/shared/mods/HD.png "Ikon mod Hidden (HD) mod")
+
+*Untuk melihat daftar keseluruhan dari semua mod, kunjungi: [Game Modifier](/wiki/Game_modifier)*\
+*Berbeda konteks dengan [Fade In (mod)](/wiki/Game_modifier/Fade_In) ataupun [Flashlight (mod)](/wiki/Game_modifier/Flashlight).*
+
+## Tentang
+
+- Singkatan: HD
+- Tipe: Peningkat Kesulitan
+- Pengali skor:
+  - ![][o!s] ![][o!t] ![][o!c]: 1.06x
+  - ![][o!m]: 1.00x
+- Tombol shortcut default: `F`
+  - Tombol shortcut default ([osu!mania](/wiki/Game_mode/osu!catch)): `F` `F` atau `Shift`+`F`
+- Caption:
+  - ![][o!s]: `Approach circle serta not-not akan memudar/menghilang perlahan-lahan, namun nilai maksimal Anda akan sedikit bertambah.`
+  - ![][o!t]: `Not-not akan menghilang perlahan sebelum anda mengenainya!`
+  - ![][o!c] `Approach circle serta not-not akan memudar/menghilang perlahan-lahan, namun nilai maksimal Anda akan sedikit bertambah.`
+  - ![][o!m]: `Not-not akan menghilang perlahan sebelum anda mengenainya!`
+- Mode-mode permainan yang sesuai: ![][o!s] ![][o!t] ![][o!c] ![][o!m]
+- Variasi (osu!mania): [Fade In](/wiki/id/Game_modifier/Fade_In)
+
+## Deskripsi
+
+Mod **Hidden** adalah sebuah [game modifier](/wiki/id/Game_modifier) yang meningkatkan kesulitan sebuah beatmap dengan menghilangkan approach circle dan menyebabkan [hit objek](/wiki/id/Hit_object) untuk memudar setelah muncul di layar.
```

Comment:
```suggestion
Mod **Hidden** adalah sebuah [game modifier](/wiki/Game_modifier) yang meningkatkan kesulitan sebuah beatmap dengan menghilangkan approach circle dan menyebabkan [objek ketukan](/wiki/Hit_object) memudar setelah muncul di layar.
```

Reply:
actually nvm, the direct of language are automated
perhaps, you can change the commit to the similar case